### ISSUE TITLE
[Accessibility] “Reduce Transparency” accessibility option makes menus unreadable

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#5425]
Logged via : https://bugzilla.mozilla.org/show_bug.cgi?id=1579580

User Agent: Mozilla/5.0 (iPhone; CPU iPhone OS 12_4_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) FxiOS/18.2b15817 Mobile/15E148 Safari/605.1.15

Steps to reproduce:

In iOS: Settings - General - Accessibility - Reduce Transparency
In FF: Settings - Display - Dark

Actual results:

Menus become too faint to read.

Expected results:

Menus should have reasonable contrast

![image](https://user-images.githubusercontent.com/32130829/64615020-58f48400-d3e2-11e9-96aa-a4de5ea1c86e.png)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2162)

**Comment by nick130586** [comment]
Not reproducible on iOS 12 and 13 simulators

**Comment by SimonBasca** [comment]
This issue is no longer reproducible on latest main 1d94cfa10d41a5faf1960e7e90ab7805e605b121 and latest v103.0 (13910)
